.cursor/skills/waiting-for-build/scripts/fetch-build.py

```
@@ -0,0 +1,309 @@
+            "Run once to set up:\n"
+            "  slackdump workspace new -no-encryption -token <xoxc-...> -cookie <xoxd-...> redhat\n\n"
+            "Or add to .mcp.json → mcpServers.slack.env:\n"
+            "  SLACK_TOKEN=xoxc-...\n"
+            "  SLACK_COOKIE=xoxd-...\n"
+            "  SLACK_CHANNEL_ID=C09DS44AQ65\n"
+            "  SLACKDUMP_WORKSPACE=redhat",
+            file=sys.stderr,
+        )
+        sys.exit(1)
+
+    cookie_file = Path(tempfile.mktemp(suffix=".txt"))
```

Ruff (0.15.15)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)